websites/C/CSGO Stash/presence.ts

```diff
@@ -0,0 +1,182 @@
+    presenceData.state = `Current lowest Steam price:
+    ${
+      document.querySelector(
+        "body > div.container.main-content > div.row.text-center > div.col-md-8.col-widen > div > div:nth-child(2) > div.col-md-6.col-md-6-collapse-top-margin > div:nth-child(2) > a > span.pull-right"
+      ).textContent
+    }`;
+    presenceData.buttons = [{ label: "View Patch", url: document.URL }];
+  } else if (path === "/graffiti") presenceData.details = "Browsing graffitis";
+  else if (path.startsWith("/graffiti/collection")) {
+    presenceData.details = "Browsing graffiti collection";
+    presenceData.state = shortTitle;
+  } else if (path.startsWith("/graffiti/tournament")) {
+    presenceData.details = "Browsing tournament graffitis";
+    presenceData.state = shortTitle;
+  } else if (path.startsWith("/graffiti/box")) {
+    presenceData.details = `Viewing graffiti box: ${shortTitle}`;
+    presenceData.state = `Current lowest Steam price:
+    ${
+      document.querySelector(
+        "body > div.container.main-content > div:nth-child(3) > div > div:nth-child(1) > div > a.btn.btn-default.market-button-item"
+      ).textContent
+    }`;
+    presenceData.buttons = [{ label: "View Graffiti Box", url: document.URL }];
+  } else if (path.startsWith("/graffiti")) {
+    presenceData.details = `Viewing graffiti: ${shortTitle}`;
+    presenceData.state = `Current lowest Steam price:
+    ${
+      document.querySelector(
+        "body > div.container.main-content > div.row.text-center > div.col-md-8.col-widen > div > div:nth-child(2) > div.col-md-6.col-md-6-collapse-top-margin > div:nth-child(2) > a > span.pull-right"
+      ).textContent
+    }`;
+    presenceData.buttons = [{ label: "View Graffiti", url: document.URL }];
+  } else if (path === "/items") {
+    presenceData.details = "Browsing items";
+    presenceData.smallImageKey = "items";
+    presenceData.smallImageText = "Viewing items";
+  } else if (path.startsWith("/item")) {
+    presenceData.details = `Viewing item: ${shortTitle}`;
+    presenceData.state = `Current lowest Steam price:
+    ${
+      document.querySelector(
+        "body > div.container.main-content > div.row.text-center > div.col-md-8.col-widen > div > div:nth-child(2) > div.col-md-6.col-md-6-collapse-top-margin > div:nth-child(2) > a > span.pull-right"
+      ).textContent
+    }`;
+    presenceData.smallImageKey = "items";
+    presenceData.smallImageText = "Viewing items";
+    presenceData.buttons = [{ label: "View Item", url: document.URL }];
+  } else if (path.indexOf("google-search") > -1) {
+    presenceData.details = "Searching for:";
+    presenceData.state = document.querySelector(
+      "#___gcse_0 > div > div > div > div.gsc-wrapper > div.gsc-resultsbox-visible > div > div > div.gcsc-more-maybe-branding-root > a > div > span.gcsc-find-more-on-google-query"
+    ).textContent;
+  } else if (document.location.href === "https://csgostash.com")
```

Comment:
This is always true
```suggestion
  } else
```

Reply:
@ImRodry not quite
It would not return as the case above when browsing  different sites
![image](https://user-images.githubusercontent.com/77577746/146646594-7936a5ea-3196-4858-82cd-c39b851620d1.png)

Reply:
![image](https://user-images.githubusercontent.com/77577746/146646612-ddaa70e9-c006-4134-8163-22f3df3427d6.png)
This is the home page case as above

Reply:
If I recall correctly
`.href` also returns slashes and branches in the site URL so the usage of `.href` there is justified